opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -137,10 +147,10 @@ fn benchmark_with_noop_layer(c: &mut Criterion, enabled: bool, bench_name: &str)
         c.bench_function(bench_name, |b| {
             b.iter(|| {
                 error!(
-                    name = "CheckoutFailed",
+                    name : "CheckoutFailed",
                     book_id = "12345",
                     book_title = "Rust Programming Adventures",
-                    "Unable to process checkout."
+                    message = "Unable to process checkout."
```

Comment:
Why have we removed `message` here?

Reply:
we added message, to make it consistent with rest of benchmarks. 
(message is preferred as it allows one less dyn dispatch: https://github.com/open-telemetry/opentelemetry-rust/pull/2689/files)